### PR TITLE
Changed primary key for Set and Spl to be set_id and spl_id respectively

### PR DESCRIPTION
**Pull request by jrlegrand**
Change Set and Spl models to use their ids as primary keys.  Tried doing the same for Ndc, but there are SPLs that have duplicate NDCs within them, so this is not possible without modifying Scrapy functionality.  Known bug = going to an Ndc in the API that is duplicate returns error.

### CODE REVIEW

**Review by yevgenybulochnik — APPROVED** [review]
Looks good, I am going to approve for now to keep things moving. I think it would be a good idea to remove the lookup field since this is introducing a known bug, but not a big deal at this point. We should put up an issue that outlines this after this is merged, unless you want to adjust that in this branch. 

Running the full 1.5g zip through scrapy gives me this output:
```
{'downloader/exception_count': 3,
 'downloader/exception_type_count/builtins.FileNotFoundError': 3,
 'downloader/request_bytes': 1267895,
 'downloader/request_count': 4388,
 'downloader/request_method_count/GET': 4388,
 'downloader/response_bytes': 774569612,
 'downloader/response_count': 4385,
 'downloader/response_status_count/200': 4385,
 'elapsed_time_seconds': 190.898982,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2020, 9, 4, 14, 3, 56, 3653),
 'log_count/DEBUG': 4388,
 'log_count/ERROR': 2,
 'log_count/INFO': 13,
 'memusage/max': 215248896,
 'memusage/startup': 68292608,
 'response_received_count': 4385,
 'retry/count': 2,
 'retry/max_reached': 1,
 'retry/reason_count/builtins.FileNotFoundError': 2,
 "robotstxt/exception_count/<class 'FileNotFoundError'>": 1,
 'robotstxt/request_count': 1,
 'scheduler/dequeued': 4385,
 'scheduler/dequeued/memory': 4385,
 'scheduler/enqueued': 4385,
 'scheduler/enqueued/memory': 4385,
 'start_time': datetime.datetime(2020, 9, 4, 14, 0, 45, 104671)}
```
You may notice the 2 errors, this is also a bug that I will push up a PR for, other than the NDC duplicates looks like its working pretty well.

File: api/dailymed/views.py [hunk 1]
```diff
@@ -8,18 +8,16 @@ class SetViewSet(mixins.RetrieveModelMixin,
...
-    lookup_field = 'name'
 
 class NdcViewSet(mixins.RetrieveModelMixin,
                     mixins.ListModelMixin,
                     viewsets.GenericViewSet):
     queryset = Ndc.objects.all()
     serializer_class = NdcSerializer
-    lookup_field = 'value'
\ No newline at end of file
+    lookup_field = 'ndc'
```
[the remark this answers is not in the document]
Comment (yevgenybulochnik) [reply api/dailymed/views.py:23]: Actually do you have the spl file names which cause this to break

[the remark this answers is not in the document]
Comment (jrlegrand) [reply api/dailymed/views.py:23]: The two XML documents are below:

- 3c8e9c87-0475-444b-bbe7-99620519a581.xml
- 8d4d72be-638b-11ea-918e-832cfc2ca371.xml

I pulled this from the API for one of them:
```javascript
GET /spl/8d4d72be-638b-11ea-918e-832cfc2ca371/
HTTP 200 OK
Allow: GET, HEAD, OPTIONS
Content-Type: application/json
Vary: Accept
{
    "id": "8d4d72be-638b-11ea-918e-832cfc2ca371",
    "set": "http://192.168.1.12:8000/set/2524b253-069e-4028-819c-361b888df110/",
    "ndcs": [
        "http://192.168.1.12:8000/ndc/50458-178-00/",
        "http://192.168.1.12:8000/ndc/50458-178-15/",
        "http://192.168.1.12:8000/ndc/50458-178-00/",
        "http://192.168.1.12:8000/ndc/50458-178-20/",
        "http://192.168.1.12:8000/ndc/50458-178-00/",
        "http://192.168.1.12:8000/ndc/50458-178-12/",
        "http://192.168.1.12:8000/ndc/50458-178-28/",
        "http://192.168.1.12:8000/ndc/50458-178-06/",
        "http://192.168.1.12:8000/ndc/50458-176-00/",
        "http://192.168.1.12:8000/ndc/50458-176-15/",
        "http://192.168.1.12:8000/ndc/50458-176-28/",
        "http://192.168.1.12:8000/ndc/50458-176-06/"
    ]
}
```

**Comment by jrlegrand** [comment]
Sounds good - I will create an issue about the duplicate NDCs.  I think we either:
a.) don't use `lookup_field`
or
b.) figure out whether it is a good idea to de-duplicate these NDCs within the SPL

PS - I clearly need a better dev machine if you ran through this in 190s.

**Pull request merged**